src/Libraries/Nop.Services/Orders/OrderProcessingService.cs

```diff
@@ -3211,28 +3211,32 @@ public virtual async Task VoidOfflineAsync(Order order)
         /// </summary>
         /// <param name="order">The order</param>
         /// <returns>A task that represents the asynchronous operation</returns>
-        public virtual async Task ReOrderAsync(Order order)
+        public virtual async Task<IList<string>> ReOrderAsync(Order order)

```

Comment:
@mike-warren-dev Update summary comment of this method since its signature is changed.

Reply:
I've updated this in both the Service and the Interface.